frontend/apps/marketing/src/components/editorialCard/EditorialCard.tsx

```diff
@@ -0,0 +1,95 @@
+import {EntryFields} from 'contentful';
+import {useMemo} from 'react';
+
+import EditorialCard, {
+  EDITORIAL_CARD_LAYOUTS,
+} from '@code-dot-org/component-library/cms/editorialCard';
+
+import {fontAwesomeV6BrandIconsMap} from '@/components/common/constants';
+import {LinkEntry} from '@/types/contentful/entries/Link';
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+export enum EDITORIAL_CARD_CONTENTFUL_LAYOUTS {
+  HORIZONTAL_WITH_IMAGE = 'horizontal_with_image',
+  VERTICAL_WITH_IMAGE = 'vertical_with_image',
+  VERTICAL_WITH_ICON = 'vertical_with_icon',
+}
+
+export interface EditorialCardContentfulProps {
+  layoutOpt: EDITORIAL_CARD_CONTENTFUL_LAYOUTS;
+  heading: EntryFields.Text;
+  text: EntryFields.Text;
+  iconName?: EntryFields.Text;
+  image?: ExperienceAsset | string;
```

Comment:
I'm a little confused by the types here. If `image` can be an ExperienceAsset and a string, then line 49 would have a src reference to an ExperienceAsset shape which isn't valid (I assume this is why you're casting?)

If Image is only ever a string, I think we should define that here and remove ExperienceAsset?